src/components/Toolbar.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <ion-header :translucent="true">
+    <ion-toolbar>
+      <div id="title-bar" >
```

Comment:
```suggestion
      <div class="title-bar" >
```

Just some semantics, only use `id` if it's really unique throughout the entire app AND you want to access it in javascript. If the former doesn't apply, opt for using css classes.